source/core/types/ut_test.tpb

```diff
@@ -54,6 +54,9 @@ create or replace type body ut_test as
 
     a_listener.fire_before_event(ut_utils.gc_test,self);
     self.start_time := current_timestamp;
+    
+    -- report to application_info
+    ut_utils.set_action(self.object_owner||'.'||self.object_name||'.'||self.name);
```

Comment:
I would use: `set_client_info` as it is 64 bytes. We should probably only put `package_name.procedure_name` so that it fits.
```
BEGIN
  DBMS_APPLICATION_INFO.set_module(ut.version, 'before_all');
  DBMS_APPLICATION_INFO.set_client_info(self.object_name||'.'||self.name);
END;
/
```